src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -669,4 +670,35 @@ public static boolean isInt(String str) {
         return isInt;
     }
 
+    /**
+     * Counts empty lines before given.
+     * @param lineNo
+     *        Line number of current import.
```

Comment:
Please update this parameter description. This is a generic method. What is `current import`?

Reply:
Updated parameter description to ```Line number of the given line, 1-based.```

Reply:
It seems the word ```1-based``` does not pass the CI spell check. Let's try ```one-based```.